src/plugins/lua/bindings/lua_ImGuiCustom.cpp

```diff
@@ -99,6 +102,39 @@ void RegisterBindings_ImGuiCustom(sol::table& ImGui)
 			[](mq::imgui::ConsoleWidget* pThis, const ImVec4& col, std::string_view text) { pThis->AppendText(text, MQColor(col), false); }
 		)
 	);
-}
 
+	// Widgets: ZepInputBox
+	ImGui.new_usertype<mq::imgui::ImGuiZepEditor>(
+		"TextEditor", sol::factories(&mq::imgui::ImGuiZepEditor::Create),
+		"Render", sol::overload(
+			[](mq::imgui::ImGuiZepEditor* pThis) { pThis->Render(); },
+			[](mq::imgui::ImGuiZepEditor* pThis, ImVec2 displaySize) { pThis->Render(displaySize); }
+		),
+		"Clear", &mq::imgui::ImGuiZepEditor::Clear,
+		"SetSyntax", [](mq::imgui::ImGuiZepEditor* pThis, const char* syntaxName) { pThis->SetSyntaxProvider(syntaxName); },
+		"text", sol::property(
+			[](mq::imgui::ImGuiZepEditor* pThis) -> std::string { size_t bufferSize = pThis->GetTextLength(); std::string text; text.reserve(bufferSize); pThis->GetText(text); return text; }
+			, &mq::imgui::ImGuiZepEditor::SetText),
+		"fontSize", sol::property(&mq::imgui::ImGuiZepEditor::GetFontSize, &mq::imgui::ImGuiZepEditor::SetFontSize),
+		"flags", sol::property(&mq::imgui::ImGuiZepEditor::GetWindowFlags, &mq::imgui::ImGuiZepEditor::SetWindowFlags)
```

Comment:
aren't there other kinds of flags? Wouldn't windowFlags be more appropriate?

Reply:
Fixed.